addons/website_sale/static/src/js/cart.js

```diff
@@ -1,17 +1,53 @@
 /** @odoo-module **/
 
 import publicWidget from "@web/legacy/js/public/public_widget";
+import { rpc } from "@web/core/network/rpc";
```

Comment:
should not we use single quotes in js files as well?

Reply:
I've personally decided to stop fighting against js team on that topic, and to follow the current standard of the file I'm working in.